spot-contracts/contracts/_utils/BondHelpers.sol

```diff
@@ -41,18 +42,13 @@ library BondHelpers {
 
     /// @notice Given a bond, retrieves all of the bond's tranches.
     /// @param b The address of the bond contract.
-    /// @return The tranche data.
-    function getTranches(IBondController b) internal view returns (BondTranches memory) {
-        BondTranches memory bt;
-        uint8 trancheCount = b.trancheCount().toUint8();
-        bt.tranches = new ITranche[](trancheCount);
-        bt.trancheRatios = new uint256[](trancheCount);
-        // Max tranches per bond < 2**8 - 1
-        for (uint8 i = 0; i < trancheCount; i++) {
-            (ITranche t, uint256 ratio) = b.tranches(i);
-            bt.tranches[i] = t;
-            bt.trancheRatios[i] = ratio;
+    /// @return bt The bond's tranche data.
+    function getTranches(IBondController b) internal view returns (BondTranches memory bt) {
+        if (b.trancheCount() != 2) {
+            revert UnacceptableTrancheLength();
         }
+        (bt.tranches[0], bt.trancheRatios[0]) = b.tranches(0);
+        (bt.tranches[1], bt.trancheRatios[1]) = b.tranches(1);
```

Comment:
I think we don't need the return statement anymore, with the new declaration syntax.

Reply:
I prefer the explicit return